tests/backup/backup_test.go

```diff
@@ -2722,6 +2722,292 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 	})
 })
 
+func GetbackupScheduleUID(backupName string, orgID string, ctx context.Context) (string, error) {
+	backupDriver := Inst().Backup
+	bkpUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
+		Name:  backupName,
+		Uid:   bkpUid,
+		OrgId: orgID,
+	}
+
+	resp, err := backupDriver.InspectBackupSchedule(ctx, backupSchInspectRequest)
+	log.FailOnError(err, "Failed for req - %v", backupSchInspectRequest)
+	log.InfoD("response here is  %v", resp)
+	log.InfoD("resp.GetBackupSchedule().GetUid() - %v ", resp.GetBackupSchedule().GetUid())
+	scheduleBackupyUid := resp.GetBackupSchedule().GetUid()
+
+	return scheduleBackupyUid, err
+
+}
+
+// Cluster backup share toggle
+var _ = Describe("{ClusterBackupShareToggle}", func() {
+	userName := "testuser133"
+	firstName := "firstName133"
+	lastName := "lastName133"
+	email := "testuser133@cnbu.com"
+	password := "Password1"
```

Comment:
Warning: This is public repo, we should be hardcoding credentials in our code

Reply:
These are just creds for the user which is create during test. I don't think this should be a problem.

Reply:
I would at least avoid using similar passwords that we use in our private infra ^^

Reply:
Maybe we should create some configmap and pass those there instead of hardcoding it, this way if something changes, we don't have to add modifications to code and just deploy new configmap